apollo-api/src/main/java/com/apollostack/api/graphql/ResponseStreamReader.java

```diff
@@ -0,0 +1,57 @@
+package com.apollostack.api.graphql;
+
+import java.io.IOException;
+import java.util.List;
+
+/** TODO java doc **/
+public interface ResponseStreamReader {
```

Comment:
it should be as it used in converter module in com.apollostack.converter.pojo.ApolloResponseBodyConverter